pkg/printer/checkstyle.go

```diff
@@ -0,0 +1,60 @@
+package printer
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/get-woke/woke/pkg/result"
+	"github.com/get-woke/woke/pkg/rule"
+)
+
+// Checkstyle is a Checkstyle printer meant for use by a Checkstyle annotation
+type Checkstyle struct {
+	writer io.Writer
+}
+
+// NewGitHubActions returns a new GitHubActions printer
+func NewCheckstyle(w io.Writer) *Checkstyle {
+	return &Checkstyle{writer: w}
+}
+
+func (p *Checkstyle) PrintSuccessExitMessage() bool {
+	return true
+}
+
+// Print prints in the format for Checkstyle
+// https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message
```

Comment:
This comment looks copied from the GitHubActions printer. Can you update it with relevant details for the checkstyle printer? A link to their documentation that explains this format would be helpful.